001-core/updates.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Automattic\VIP\Core\Updates;
+
+/**
+ * Show plugin update notices on the plugins page.
+ *
+ * This is basically a copy of Core's wp_plugin_update_rows() function.
+ * The only difference is that we use the manage_options cap instead of update_plugins.
+ * Because no one on VIP has update_plugins, core's notices are never displayed.
+ * With this change, we make them visible.
+ */
+function show_plugin_update_notices() {
+	if ( ! current_user_can( 'manage_options' ) ) {
```

Comment:
For correctness, should use the `activate_plugins` cap.

```suggestion
 * The only difference is that we use the activate_plugins cap instead of update_plugins.
 * Because no one on VIP has update_plugins, core's notices are never displayed.
 * With this change, we make them visible.
 */
function show_plugin_update_notices() {
	if ( ! current_user_can( 'activate_plugins' ) ) {
```

Reply:
Wouldn't `update_plugins` ([noted here](https://github.com/WordPress/WordPress/blob/3da046e1bce8408a1b6cd08e03eaf69a6bb506cd/wp-includes/capabilities.php#L439)) by more accurate - i.e. only show notices about plugin updates to those who have the capability update plugins? Activating !== Updating.